lms/static/sass/discussion/views/_thread.scss

```diff
@@ -9,197 +9,301 @@
-    @extend %t-copy-sub1;
-    // clear: both; //TO-DO: confirm that removing this is ok for all cases of discussion posts.
-  }
 
-  // post context
-  .post-context {
-    @extend %t-copy-sub2;
-    margin-top: $baseline;
-    color: $gray-d1;
-
-    // CASE: no courseware context or cohort visibility rules
-    &:empty {
-      display: none;
+    // post body
+    .post-body {
+        @extend %t-copy-sub1;
+        // clear: both; //TO-DO: confirm that removing this is ok for all cases of discussion posts.
     }
 
-    // post visibility - cohorts
-    .group-visibility-label {
-      margin-top: ($baseline/4);
+    // post context
+    .post-context {
+        @extend %t-copy-sub2;
+        margin-top: $baseline;
+        color: $gray-d1;
+
+        // CASE: no courseware context or cohort visibility rules
+        &:empty {
+            display: none;
+        }
+
+        // post visibility - cohorts
+        .group-visibility-label {
+            margin-top: ($baseline/4);
+        }
     }
-  }
 }
 
 // Layout control for discussion modules that does not apply to the discussion board
 .discussion-module {
-  .discussion-thread {
+    .btn-brand {
+        @include blue-button;
+        display: inline-block;
+        padding-bottom: ($baseline/10);
+        height: 37px;
+
+        &:hover, &:focus {
+            border-color: #222;
+        }
+    }
+}
+
+// Styling for discussion threads
+.discussion-thread {
+    padding: 0;
+    margin-bottom: $baseline;
+    @include transition(all .25s linear 0s);
+
+    p {
+        margin-bottom: 0;
+    }
+
     .thread-main-wrapper, .thread-responses-wrapper {
-      padding: $baseline;
+        padding: $baseline;
     }
-  }
 
-  .btn-brand {
-    @include blue-button;
-    display: inline-block;
-    padding-bottom: ($baseline/10);
-    height: 37px;
+    .discussion-article {
+        @include transition(all .2s linear 0s);
+        border: 1px solid $forum-color-border;
+        border-radius: 3px;
+        min-height: 0;
+        background: $forum-color-background;
+        box-shadow: 0 1px 0 $shadow;
+        @include transition(all .2s linear 0s);
+
+        .thread-wrapper {
+            @include border-radius(3px, 3px, 0, 0);
+            position: relative;
+            overflow-x: hidden;
+            overflow-y: auto;
+            max-height: 600px;
+            background-color: $forum-color-background;
+
+            .discussion-post {
+
+                .inline-comment-count {
+                    @extend %ui-depth2;
+                    position: relative;
+                    float: right;
+                    display: block;
+                    height: 27px;
+                    margin-top: 6px;
+                    margin-right: 8px;
+                    padding: 0 8px;
+                    border-radius: ($baseline/4);
+                    font-size: 12px;
+                    font-weight: 400;
+                    line-height: 25px;
+                    color: #888;
```

Comment:
I haven't done a sweep of all the colors yet. I want to move them all to be using variables, but I'm not going to complete that with this PR.